### PR TITLE
docs: add some docs around a schema value for new installs

### PR DESCRIPTION
**Pull request by slim-bean**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

File: docs/sources/operations/storage/schema/_index.md [hunk 1]
```diff
@@ -14,6 +14,33 @@ Loki uses the defined schemas to determine which format to use when storing and
 
 Use of a schema allows Loki to iterate over the storage layer without requiring migration of existing data.
 
+## New Loki installs
+``
```
Comment (DylanGuedes) [inline docs/sources/operations/storage/schema/_index.md:18]:
```suggestion

```

File: docs/sources/operations/storage/schema/_index.md [hunk 2]
```diff
@@ -33,19 +60,19 @@ Here are items to consider when changing the schema; if schema changes are not d
...
-          object_store: gcs
-          schema: v13
-          store: tsdb
+  configs:
+    - from: "2020-07-31"
+      index:
+        period: 24h
+        prefix: loki_ops_index_
+      object_store: gcs
+      schema: v11
```
Comment (JStickler) [inline docs/sources/operations/storage/schema/_index.md:69]: Should this line be here?  You've also got the schema defined at line 76.

File: docs/sources/operations/storage/schema/_index.md [hunk 3]
```diff
@@ -14,6 +14,33 @@ Loki uses the defined schemas to determine which format to use when storing and
...
+
+| Property     | Description                                                                                                                                            |
+|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| from         | for a new install, this must be a date in the past, use a recent date.                                                                                 |
+| object_store | s3, azure, gcs, alibabacloud, bos, cos, swift, filesystem, or a named_store (see [StorageConfig]({{< relref "../../../configure#storage_config" >}})). |
+| store        | `tsdb` is the current and only recommended value for store.                                                                                            |
+| schema       | `v13` is the most recent schema and recommended value.                                                                                                 |
+| prefix:      | any value with out spaces is acceptable.                                                                                                               |
```
Comment (DylanGuedes) [inline docs/sources/operations/storage/schema/_index.md:40]:
```suggestion
| prefix:      | any value without spaces is acceptable.                                                                                                               |
```

File: docs/sources/operations/storage/schema/_index.md [hunk 4]
```diff
@@ -14,6 +14,33 @@ Loki uses the defined schemas to determine which format to use when storing and
...
+      schema: v13
+      index:
+        prefix: index_
+        period: 24h
+```
+
+
+| Property     | Description                                                                                                                                            |
+|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| from         | for a new install, this must be a date in the past, use a recent date.                                                                                 |
```
Comment (DylanGuedes) [inline docs/sources/operations/storage/schema/_index.md:36]:
```suggestion
| from         | for a new install, this must be a date in the past, use a recent date. Format is YYYY-MM-DD.                                                                                 |
```

**Review by DylanGuedes — APPROVED** [review]
lgtm few nits

File: docs/sources/operations/storage/schema/_index.md [hunk 5]
```diff
@@ -33,19 +60,19 @@ Here are items to consider when changing the schema; if schema changes are not d
...
+      object_store: gcs
+      schema: v11
+      store: tsdb
+    - from: "2022-01-20"
+      index:
+        period: 24h
+        prefix: loki_ops_index_
+      object_store: gcs
+      schema: v13
+      store: tsdb
```
Comment (JStickler) [inline docs/sources/operations/storage/schema/_index.md:77]: Copy/ paste error?
Lines 68 - 70 are the same as lines 75 - 77, with the exception of the schema value.

File: docs/sources/operations/storage/schema/_index.md (hunk 1 above)
Comment (JStickler) [inline docs/sources/operations/storage/schema/_index.md:18]:
```suggestion

```

**Review by JStickler — APPROVED** [review]
[docs team]  Other than two random ticks at line 18, LGTM.

File: docs/sources/operations/storage/schema/_index.md (hunk 5 above)
Comment (slim-bean) [reply docs/sources/operations/storage/schema/_index.md:76]: actually all this was is a change to indentation to remove the tab chars and use spaces instead which I think is better for copy pasting yaml

**Pull request merged**